communitytoolkit/maui/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: .NET Multi-platform App UI (.NET MAUI) Community Toolkit
+author: bijington
+ms.author: bramin
+description: The .NET MAUI Community Toolkit is a collection of reusable elements for application development with .NET MAUI, including animations, behaviors, converters, effects, and helpers.
+ms.date: 01/14/2022
+---
+
+# .NET Multi-platform App UI (.NET MAUI) Community Toolkit
+
+The .NET MAUI Community Toolkit is a collection of reusable elements for application development with [.NET MAUI][maui-url], including animations, behaviors, converters, effects, and helpers. It simplifies and demonstrates common developer tasks when building iOS, Android, macOS and WinUI applications using [.NET MAUI][maui-url].
+
+> [!IMPORTANT]
+> The .NET MAUI Community Toolkit is currently in preview.
```

Comment:
Maybe better to use an include file, as this will be repeated on every page? Like https://github.com/dotnet/docs-maui/blob/main/docs/includes/preview-note.md

Reply:
Oh yes I love this idea! Thanks I didn't know that was possible